examples/63_Conic.html

```diff
@@ -36,12 +37,45 @@
 		              {name:"Co", type:"ConicBy5", args:["A","B","C","D","E"],color:[0,0,1],alpha:1,size:1},
 //		     {name:"l", type:"Join", args:["A", "D"]}
                       ];
-            createCindy({canvasname:"CSCanvas",
+            var cdy=createCindy({canvasname:"CSCanvas",
                         movescript:"csmove",
                         grid:1,
                         snap:true,
                         geometry:gslp});
 
+            var cases = [
+                ["parabola1", [0,0,1], [1,1,1], [2,4,1], [-2,4,1], [-1,1,1]],
```

Comment:
When instrumented, clicking on the 'parabola1' button shows a simple stroke sequence:
```javascript
moveTo(157.02540452080043,0)
quadraticCurveTo(250.49999999999997,698.9999999999992,343.97459547919953,0)
```
Adding positioning for the rotation of 'parabola1' by 45&deg; clockwise:
```javascript
 +                ["parabola2", [0, 0, 1], [0, 2, 1], [6, 2, 1], [2, 6, 1], [2, 0, 1]],
```
and then clicking on the 'parabola2' however yields a sequence consisting of 80 quadraticCurveTo calls. It would seem that this could be a simple sequence containing just one quadraticCurveTo call.

Adding positioning for a circle:
```javascript
 +                ["circle1", [6,1,1], [10,3,1], [10,9,1], [2,9,1], [2,3,1]],
```
and then clicking on the 'circle1' yields a sequence consisting of 128 quadraticCurveTo calls interspersed with two lineTo calls.
```javascript
moveTo(275.5,199.5)
quadraticCurveTo(275.5,196.43142223638486,275.65056797435346,193.3665407090727)
...
quadraticCurveTo(397.4314222363835,74.50000000000011,400.5,74.50000000000011)
lineTo(400.5,74.50000000000001)
quadraticCurveTo(403.56857776361465,74.50000000000001,406.6334592909267,74.65056797435348)
...
quadraticCurveTo(403.56857776361323,324.5,400.5,324.5)
lineTo(400.5,324.4999999999999)
quadraticCurveTo(397.43142223638347,324.4999999999999,394.36654070907263,324.3494320256465)
...
quadraticCurveTo(275.5,202.5685777636156,275.5,199.5)
closePath()
```
The use of 'lineTo' appears to be a bit over zealous.

It would seem that this should be a simple sequence containing just four quadraticCurveTo calls (or perhaps eight for very large circles) as is typically done in PDF files to no perceptible visual difference.